Header header
string reference_frame
string target_frame
float32 mag_horiz
float32 east
float32 nrth
float32 elev